src-built-in/preloads/zoom.js

```diff
@@ -191,20 +199,33 @@ const zoomConfigHandler = (err, zoom) => {
 }
 
 /**
- * Applies the zoom level from the component state.
+ * Applies the zoom level from the component state or default configuration.
  *
  * @param {*} err The error, if one occurred, from getting the zoom level from component state.
  * @param {Number} zoomLevel The zoom level saved in the component.
  */
 const getZoomLevelHandler = (err, zoomLevel) => {
 	if (err) {
 		FSBL.Clients.Logger.info("No \"fsbl-zoom\" settings found in component state", err);
-	}else if (zoomLevel != null) {
+	} else if (zoomLevel != null) {
+		FSBL.Clients.Logger.info(`Retrieved zoomLevel from state: ${zoomLevel}`, err);
 		window.fsblZoomLevel = zoomLevel;
 		setZoom(window.fsblZoomLevel);
+		window.settingInitialZoom = false;
+	} else {
+		//check for default configuration for zoom level and apply as needed
+		let defaultLevel = _.get(FSBL.Clients.WindowClient.options.customData, "foreign.components.['Window Manager'].zoomDefault");
+		if (defaultLevel) { 
+			FSBL.Clients.Logger.info(`Retrieved default zoom level from config: ${defaultLevel}`, err);
```

Comment:
I don't think `err` should be an argument here.

Reply:
fixed